imdb_django/management/commands/import_akas.py

```diff
@@ -0,0 +1,84 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.utils import IntegrityError  # Import IntegrityError
+
+from imdb_django.models import (
+    TitleAka,
+)  # Replace 'your_app_name' with your actual app name
+
+
+class Command(BaseCommand):
+    help = "Load data from TSV file into MySQL database"
+
+    def add_arguments(self, parser):
+        parser.add_argument("tsv_file", type=str, help="Path to the TSV file")
+
+    def handle(self, *args, **options):
```

Comment:
Lets Code in `handle` is quite lengthy which is hard to understand.

Leys move logic from this `handle` function into multiple custom functions

Please revisit this PR and make custom functions where there is too lengthy code in a function